homepage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Please move this inside the submissions folder

Reply:
This file has been deleted on the local repo